source/operations/concepts.rst

```diff
@@ -8,126 +8,163 @@ Core Operational Concepts
-   The whole MinIO deployment consisting of one or more Server Pools. Each
-   :mc-cmd:`~minio server HOSTNAME` argument passed to the 
-   :mc:`minio server` command represents one Server Pool:
+Link one or more server pools into a single MinIO cluster
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+A cluster refers to an entire MinIO deployment consisting of one or more Server Pools. 
 
-   .. code-block:: shell
+Consider the command below that creates a cluster consisting of two Server Pools, each with 4 :mc:`minio server` nodes and 4 drives per node for a total of 32 drives. 
 
-      minio server https://minio{1...4}.example.net/mnt/disk{1...4} \
-                   https://minio{5...8}.example.net/mnt/disk{1...4}
+.. code-block:: shell
+
+   minio server https://minio{1...4}.example.net/mnt/disk{1...4} \
+                https://minio{5...8}.example.net/mnt/disk{1...4}
                    
-                   |                    Server Pool                |
+                |                    Server Pool                |
    
-   The above example describes two Server Pools, each consisting of 4
-   :mc:`minio server` nodes with 4 drives each for a total of 32 drives. MinIO 
-   always stores each unique object and all versions of that object on the 
-   same Server Pool.
+Within a cluster, MinIO always stores each unique object and all versions of that object on the same Server Pool.
+
+A standalone cluster consists of a single Server Pool with a single :mc:`minio server` node. 
+Standalone clusters are best suited for initial development and evaluation. 
+MinIO strongly recommends production clusters consist of a *minimum* of 4 :mc:`minio server` nodes in a Server Pool for proper high availability and durability guarantees.
+
+Adjust the number of pools available in a cluster
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+When the time comes to retire or replace a server pool, :ref:`decommissioning <minio-decommissioning>` is the process of draining the objects of a pool to other active pools on the deployment.
+MinIO rewrites objects from the decommissioned pool and does not allow additional writes to the pool while it is in decommissioning.
+   
+Once started, decommissioning cannot be stopped.
+
+In a distributed MinIO deployment, you can upgrade the total available size of a :ref:`deployment <expand-minio-distributed>` or :ref:`cluster <minio-k8s-expand-minio-tenant>` by adding one or more additional server pools.
+The addition of a server pool is an expansion.
+
+.. cond:: k8s
+
+   Deploy multiple MinIO clusters within a single management console
+   ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+   In Kubernetes, a tenant is a specific instance of MinIO with its own node(s), buckets, objects, volumes, access keys, and lifecycle management rules.
+   The MinIO Operator can manage many MinIO tenants.
+
+
+How does MinIO provide availability, redundancy, and reliability?
```

Comment:
I really like this section, but again think the headers need to be more SEO focused. Spitballing:

"MinIO uses Erasure Coding for Data Redundancy and Reliability"

"MinIO implements Bitrot Healing to Protect Data At Rest"

"MinIO distributes data across erasure sets for high availability and resiliency"

"MinIO automatically heals corrupt or missing data on-the-fly"

Reply:
I need to think more in terms of SEO. Going with these suggestions for now.